config/prometheus/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
prometheus folder can be deleted. we will fetch the metrics of the operator from our own prometheus